architecture/standards/0007-java-pre-requisite.md

```diff
@@ -0,0 +1,53 @@
+# ADR-0007 - Gradle and Java as a pre-requisite
```

Comment:
```suggestion
# ADR-0007 - Gradle and Java as a prerequisite
```

Reply:
Reworked the title, no longer applicable